src/app/assets/languages/en.json

```diff
@@ -269,20 +269,22 @@
           "mode-tooltip": "Control whether state transfer happens manually on user action, which is the default, or automatically when backup locations come online. {{brandname}} can perform automatic state transfer with the ASYNC backup strategy only.",
           "mode-manual": "MANUAL",
           "mode-auto": "AUTOMATIC",
-          "transactional": "Transactional",
-          "transactional-description": "Allows {{brandname}} to participate in distributed transactions compliant with JTA.",
-          "transactional-mode": "Transactional mode",
-          "transactional-mode-tooltip": "Configure the transactional mode for the cache. Transactional caches are not shared across nodes and are not replicated. Transactional caches are useful for high-performance transactional data stores.",
+          "transactional": "Transactions",
+          "transactional-description": "Configure {{brandname}} to participate in JTA-compliant distributed transactions.",
+          "transactional-mode": "Transaction mode",
+          "transactional-mode-tooltip": "Configure the mode that {{brandname}} uses when carrying out ACID (atomicity, consistency, isolation, durability) transactions to ensure the cache state is consistent.",
```

Comment:
@pruivo Do you think it's too much to put the ACID reference here?

Reply:
probably :) ACID property is not much relevant for transaction mode.

Reply:
Thanks, I've removed it.